resolwe_bio/tools/summarize_expression_ncRNA.py

```diff
@@ -31,8 +31,8 @@
 
 gene_id_re = re.compile(r'ID=([\w\-\.]*)')
 gene_name_re = re.compile(r'oId=([\w\-\.]*)')
-class_code_re = re.compile(r'class_code=([\w\-\.]*)')
 parent_id_re = re.compile(r'Parent=([\w\-\.]*)')
+class_code_re = re.compile(r'class_code=([\w\-\.]*)')
```

Comment:
@mstajdohar, I think you are right since [this check](https://github.com/PyCQA/pycodestyle/blob/d12e2ea39745bd3bb9e819d7342e4c1d73dbb008/pycodestyle.py#L293-L294) only looks if the previous (logical) line starts with `def` or `class`.

@dblenkus, will you report an issue [@PyCGA/pycodestyle](https://github.com/PyCQA/pycodestyle/issues)?